entoas/generator.go

```diff
@@ -678,7 +687,12 @@ func reqBody(n *gen.Type, op Operation) (*ogen.RequestBody, error) {
 		if err != nil {
 			return nil, err
 		}
-		if (a != nil && !a.ReadOnly) && (op == OpCreate || !f.Immutable) {
+
+		if a.ReadOnly || a.Skip {
```

Comment:
The check for `a != nil` is required, isn't it?

Reply:
Why need check? `a` can be nil only if `FieldAnnotation` return err. But this case we handle on [684-687](https://github.com/ent/contrib/pull/329/files#diff-8a343f04c5df9bc0b27ce52ca7989745e813d4779329f49f69ca18d74963c993R684-R687)